src/styles/blocks/reviews.scss

```diff
@@ -0,0 +1,69 @@
+.reviews {
+
+  &__title {
+    margin-top: 64px;
+    margin-bottom: 53px;
+
+    @media (min-width: $breakPointPc) {
+      margin-top: 152px;
+      margin-bottom: 64px;
+    }
+  }
+
+  &__review-wrapper {
+    display: grid;
+    grid-template-columns: 280px;
```

Comment:
abundant, you've already set width for each review card

```suggestion
```

Reply:
Thanks, i forgot what to default grid add new rows and not columns.